app/views/customers/show.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>Customer: <%= @customer.name %></h1>
+<section class="item_list">
+  <h3>Items:</h3>
+  <% @customer.items.each do |item| %>
+    <li><%= item.name %>, Price: <%= item.price %>, Supermarket: <%= item.supermarket.name %></li>
+  <% end %>
+</section>
+<h3>Add Item:</h3>
+<%= form_with url: "/customers/#{@customer.id}", method: :post, data: {turbo: false} do |f| %>
```

Comment:
A more RESTful path here could include `items` just to indicate that you're not POSTing a new customer but rather a relationship between a customer and an item. Consider: "/customers/:id/items"